app/gui/src/dashboard/components/Inputs/ComboBox/ComboBox.tsx

```diff
@@ -81,6 +81,16 @@ export interface ComboBoxProps<Schema extends TSchema, TFieldName extends FieldP
   readonly toTooltip?: (item: FieldValues<Schema>[TFieldName]) => string
   /** Hide the `x` button to disable resetting the input. */
   readonly noResetButton?: boolean
+  readonly addonStart?:
+    | InputProps<Schema, TFieldName, string>['addonStart']
+    | ((
+        item: FieldValues<Schema>[TFieldName],
+      ) => InputProps<Schema, TFieldName, string>['addonStart'])
+  readonly addonEnd?:
+    | InputProps<Schema, TFieldName, string>['addonStart']
+    | ((
+        item: FieldValues<Schema>[TFieldName],
+      ) => InputProps<Schema, TFieldName, string>['addonStart'])
```

Comment:
Shouldn it be
```suggestion
    | InputProps<Schema, TFieldName, string>['addonEnd']
    | ((
        item: FieldValues<Schema>[TFieldName],
      ) => InputProps<Schema, TFieldName, string>['addonEnd'])
```

?